docs/usage.md

```diff
@@ -1,17 +1,21 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+|----------------------|
 
 # Usage
 
-- Navigate to the newly created dashboard **SOC Management Dashboard**. 
-- The Dashboard provides an overall view of an investigation scenario in a SOC. 
+- Navigate to the newly created dashboard **SOC Management Dashboard**.
 
+- The Dashboard provides an overall view of an investigation scenario in a SOC.
 
 - Below is an example of the Dashboard with Demo records
 
     ![SOC Management Dashboard](res/soc_mngt_dashboard.png)

```

Comment:
I feel, it will be very lucrative and exciting for an end user to see the sample dashboard in overview (landing page). Can we move this image to overview section then?

Note: Please make sure that image is uptodate as well

Reply:
@cs-abhishek-shukla